vsintegration/src/FSharp.Editor/QuickInfo/Views.fs

```diff
@@ -78,43 +78,45 @@ type internal QuickInfoViewProvider
             t.Background <- Media.SolidColorBrush(Media.Color.FromRgb(color.R, color.G, color.B))
             t
 
-        let inlines = 
-            seq { 
-                for taggedText in content do
-                    let run = Documents.Run taggedText.Text
-                    let inl =
-                        match taggedText with
-                        | :? Layout.NavigableTaggedText as nav when navigation.IsTargetValid nav.Range ->                        
-                            let h = Documents.Hyperlink(run, ToolTip = secondaryToolTip nav.Range)
-                            h.Click.Add <| navigateAndDismiss nav.Range
-                            h :> Documents.Inline
-                        | _ -> run :> _
-                    DependencyObjectExtensions.SetTextProperties (inl, layoutTagToFormatting taggedText.Tag)
-                    yield inl
-            }
-
-        let createTextLinks () =
-            let tb = TextBlock(TextWrapping = TextWrapping.Wrap, TextTrimming = TextTrimming.None)
-            DependencyObjectExtensions.SetDefaultTextProperties(tb, formatMap.Value)
-            tb.Inlines.AddRange inlines
-            if tb.Inlines.Count = 0 then tb.Visibility <- Visibility.Collapsed
-            tb.Resources.[typeof<Documents.Hyperlink>] <- getStyle()
-            tb :> FrameworkElement
-            
-        { new IDeferredQuickInfoContent with member x.Create() = createTextLinks() }
-
-    let empty = 
-        { new IDeferredQuickInfoContent with 
-            member x.Create() = TextBlock(Visibility = Visibility.Collapsed) :> FrameworkElement }
+        let toInline (taggedText: Layout.TaggedText) =
+            let run = Documents.Run taggedText.Text
+            let inl =
+                match taggedText with
+                | :? Layout.NavigableTaggedText as nav when navigation.IsTargetValid nav.Range ->                        
+                    let h = Documents.Hyperlink(run, ToolTip = secondaryToolTip nav.Range)
+                    h.Click.Add <| navigateAndDismiss nav.Range
+                    h :> Documents.Inline
+                | _ -> run :> _
+            DependencyObjectExtensions.SetTextProperties (inl, layoutTagToFormatting taggedText.Tag)
+            inl
+
+        let tb = TextBlock(TextWrapping = TextWrapping.Wrap, TextTrimming = TextTrimming.None)
+        DependencyObjectExtensions.SetDefaultTextProperties(tb, formatMap.Value)
+        tb.Inlines.AddRange(content |> Seq.map toInline)
+        if tb.Inlines.Count = 0 then tb.Visibility <- Visibility.Collapsed
+        tb.Resources.[typeof<Documents.Hyperlink>] <- getStyle()
+        tb
+
+    let wrap (tb: TextBlock) =
+        let maxWidth = formatMap.Value.DefaultTextProperties.FontRenderingEmSize * 60.0
```

Comment:
What is 60.0 here?  If this can't be usefully factored into a well-named constant, I think a comment explaining it would be helpful.